components/prompt_based_laion_retrieval/src/main.py

```diff
@@ -82,6 +88,11 @@ def transform(
         # reorder columns
         dataframe = dataframe[["id", "source", "images_url"]]
 
+        dataframe = dataframe.astype({'id': 'string', 'source': 'string'})
+
+        print("Final dataframe:", dataframe.head(4))
+        dataframe = dataframe.reset_index(drop=True)
```

Comment:
Why are you resetting the index here?

Reply:
Because @GeorgesLorre asked me to do so :D